Mathlib/Topology/LocalHomeomorph.lean

```diff
@@ -443,6 +447,17 @@ theorem preimage_open_of_open {s : Set β} (hs : IsOpen s) : IsOpen (e.source 
   e.continuousOn.preimage_open_of_open e.open_source hs
 #align local_homeomorph.preimage_open_of_open LocalHomeomorph.preimage_open_of_open
 
+/-- A local homeomorphism is an open map on its source. -/
+lemma isOpenMapOn_source {s : Set α} (hs : IsOpen s) (hse : s ⊆ e.source) : IsOpen (e '' s) := by
+  rw [(image_eq_target_inter_inv_preimage (e := e) hse)]
+  exact e.continuous_invFun.preimage_open_of_open e.open_target hs
+
+/-- The inverse of a local homeomorphism `e` is an open map on `e.target`. -/
+lemma isOpenMapOn_target_symm {t : Set β} (ht : IsOpen t) (hte : t ⊆ e.target) :
+    IsOpen (e.invFun '' t) := by
```

Comment:
```suggestion
    IsOpen (e.symm '' t) := by
```
Normally, we only go through the coercion, i.e., we never write `e.toFun` or `e.invFun`, but rather `e` and `e.symm`. Also, this lemma should be a direct application of the previous one, no need to redo the proof.